components/Sorter.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="sorter">
+    <div class="sortHolder">
+      <a :class="active === 'All Issues' ? 'active' : ''" @click="changeFilter"
```

Comment:
probably cleaner to use this syntax here: `<div :class="{ active: active }"></div>`

https://vuejs.org/v2/guide/class-and-style.html